Userland/Libraries/LibWeb/Layout/TextNode.cpp

```diff
@@ -288,7 +288,7 @@ static ErrorOr<DeprecatedString> apply_text_transform(DeprecatedString const& st
     case CSS::TextTransform::MathAuto:
         return apply_math_auto_text_transform(string);
     case CSS::TextTransform::Capitalize: {
-        return TRY(Unicode::to_unicode_titlecase_full(string, {}, Unicode::TrailingCodePointTransformation::PreserveExisting)).to_deprecated_string();
+        return TRY(Unicode::to_unicode_titlecase_full(string, {}, Unicode::TrailingCodePointTransformation::PreserveExisting));
```

Comment:
When you switch to `String`, let's use the methods on `String` itself rather than the free helper functions:
```
string.to_titlecase();
```
(You might need to add the PreserveExisting paramter).

Reply:
Same above for the upper/lowercase transformations:
```c++
string.to_lowercase();
string.to_uppercase();
```

Reply:
And if these are the last users of those `to_unicode_*_full` methods, we can finally be rid of them :)

Reply:
There was one last user in FontEditor, I figured it was small enough of a change that I'd include it here. Removed those methods, and ported the relevant tests over :^)